### PR TITLE
update steering committee tags

### PR DESCRIPTION
**Pull request by raynamharris**
changed for former steering committee member for Adina Howe

### CODE REVIEW

**Comment by raynamharris** [comment]
Okay, @gvwilson, I think that's all the updates that need to be made for the steering committee as previously discussion in https://github.com/swcarpentry/website/issues/377#issuecomment-237258459

File: _includes/people/davis_m.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <a href="http://www.autodesk.com/">Autodesk</a>.
     His interests include scientific Python,
     web applications, computing education, diversity in tech,
-    and exploring beautiful California.
-    <br/>
-    <strong>Steering Committee (Vice-Chair)</strong>
+    and exploring beautiful California.Matt is a former
```
Comment (gvwilson) [inline _includes/people/davis_m.html:7]: Space after '.' before 'Matt'

File: _includes/people/chuang-howe_a.html [hunk 2]
```diff
@@ -3,7 +3,6 @@
     is an assistant professor in the Department of Agricultural and
     Biosystems Engineering at Iowa State University, where she uses
     skills learned from Software Carpentry to train students and study
-    microbial communities in the environment.
-    <br/>
-    <strong>Steering Committee</strong>
+    microbial communities in the environment. Adina is a former member 
+    of the Software Carpentry Foundation's Steering Committee.
```
Comment (gvwilson) [inline _includes/people/chuang-howe_a.html:7]: "of the Steering Committee" (to be consistent with others' entries).

**Comment by gvwilson** [comment]
Thanks @raynamharris - two minor corrections, then it's ready to merge.

**Comment by jduckles** [comment]
Merging then I'll fix @gvwilson's comments.

**Pull request merged**